client/angular/controllers/MainController.js

```diff
@@ -193,6 +208,9 @@
             } else {
                 $scope.displayColumns = sortedColumns;
             }
+            $scope.onscreenTweets = (backfilledColumns.reduce(function(prevColumn, curColumn) {
```

Comment:
Yes, when I initially wrote this admin view was true on mobile so it had to use the backfilled columns, but now display columns makes more sense.